CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing
 
-Demonstrating Engineering Fundamentals is core to what we do at CSE and is one of the primary values we bring to our customers, helping them to level up while collaborating on their business scenarios.
+Demonstrating Engineering Fundamentals is core to what we do in ISE and is one of the primary values we bring to our customers, helping them to level up while collaborating on their business scenarios.
 
-The purpose of this repo is to provide guidance to CSE Dev Crews with regards to engineering fundamentals. It describes recommended practices based on continuous project learnings in different areas. It can be used as a tool at the beginning of an engagement that can be shared with customers to help set the project up for success. This repo is *not* intended as a code repo, instead it provides guidance and links to appropriate sample code repos that represent good examples.
+The purpose of this repo is to provide guidance to ISE software engineers and data scientists with regards to engineering fundamentals. It describes recommended practices based on continuous project learnings in different areas. It can be used as a tool at the beginning of an engagement that can be shared with customers to help set the project up for success. This repo is *not* intended as a code repo, instead it provides guidance and links to appropriate sample code repos that represent good examples.
```

Comment:
same as earlier comment... Dev Crew is the term we typically use, and it involves more than just the engineers and data scientists, ex. PMO, TPMs, Dev Lead etc. etc. so it is much more general to say dev crew

Reply:
While I agree both SE and DS make up a Dev Crew, we changed it to this so it was explicit that is applied to both groups, not just the traditional "dev crew" consisting of SEs only.
Happy to revert to dev crew is you still prefer?

Reply:
Finally went with "ISE software engineers and data scientists (together forming a Dev Crew)" to make it clear that it applies to both populations and generically we describe them as a crew.